angular-workspace/projects/ni/nimble-angular/table-column/number-text/tests/nimble-table-column-number-text.directive.spec.ts

```diff
@@ -644,14 +644,17 @@ describe('NimbleTableColumnNumberText', () => {
         });
 
         it('can be configured with attribute binding for format', () => {
-            expect(directive.format).toBe(NumberTextFormat.roundToInteger);
-            expect(nativeElement.format).toBe(NumberTextFormat.roundToInteger);
+            expect(directive.format).toBe(NumberTextFormat.decimal);
+            expect(nativeElement.format).toBe(NumberTextFormat.decimal);
 
-            fixture.componentInstance.format = NumberTextFormat.decimal;
+            fixture.componentInstance.format = NumberTextFormat.default;
             fixture.detectChanges();
 
-            expect(directive.format).toBe(NumberTextFormat.decimal);
-            expect(nativeElement.format).toBe(NumberTextFormat.decimal);
+            // Setting the attribute to `undefined` (the value backing NumberTextFormat.default) removes
+            // the attribute, which results in the web component's property being set to `null`.
+            // See https://github.com/microsoft/fast/issues/6630.
+            expect(directive.format).toBeFalsy();
+            expect(nativeElement.format).toBeFalsy();
```

Comment:
I'm not thrilled about how I had to write this part of the test. If anyone has other suggestions, I'd love to hear them.